source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
+   .. code-block:: console
 
-.. code-block:: bash
+      {
+        "proxies":
+        {
+          "default":
+          {
+            "httpProxy": "proxy-url",
+            "httpsProxy": "proxy-url"
+          }
+        }
+      }
 
-   sudo mixer build image
+#. Set ownership and permission on the docker config directory:
 
-Mixer automatically looks for the :file:`release-image-config.json` file, but
-you can freely choose the filename. To use a different name, simply pass the
-*--template* flag when creating your image, for example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      sudo chown "$USER":"$USER" /home/"$USER"/.docker -R
+      sudo chmod g+rwx "$HOME/.docker" -R
 
-   sudo mixer build image --template path/to/file.config
+Lastly, configure proxies to allow mixer to access upstream content from behind
+a firewall. For example:
 
-By default, `ister` uses the format version of the build machine it runs on.
-Therefore, if the format you are building differs from the format of the |CL|
-OS you are building on, you must use the *--format <FORMAT_NUMBER>*
-flag. Find the current format version of your OS with the following command:
+#. Open your :file:`.bashrc` file:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   sudo cat /usr/share/defaults/swupd/format
+      nano $HOME/.bashrc
 
-Update the next mix version information
-=======================================
+#. Add your proxy values for the following:
 
-Increment the mix version number for the next mix with the following command:
+   .. code-block:: console
 
-.. code-block:: bash
+      export http_proxy="<proxy-url>"
+      export https_proxy="<proxy-url>"
+      export HTTP_PROXY="<proxy-url>"
+      export HTTPS_PROXY="<proxy-url>"
+      export no_proxy="<...>"
 
-   mixer versions update
+#. Log out and log back in for the proxies to take effect.
 
-This command automatically updates the mix version stored in the
-:file:`mixversion` file, incrementing it by 10. To increment by a different
-amount, use the *--increment* flag, for example:
+.. rst-class:: content-collapse
 
-.. code-block:: bash
+Set up a nginx web server for mixer
+===================================
 
-   mixer versions update --increment 10
+A web server is needed to host your update content. In this example, we use the
+nginx web server, which comes with |CL|.
 
-Alternatively, to set the mix version to a specific value, use the
-*--mix-version* flag, for example:
+Set up a nginx web server for mixer with the following steps:
 
-.. code-block:: bash
+#. Install the :command:`web-server-basic` bundle:
 
-   mixer versions update --mix-version 20
+   .. code-block:: bash
 
-The :command:`mixer versions update` command does not allow you to set the
-mix version to a value less than its current value. The mix version is
-expected to always increase, even if the new mix is undoing an earlier
-change.
+      sudo swupd bundle-add web-server-basic
```

Comment:
" sudo swupd bundle-add web-server-basic" --> " sudo swupd bundle-add nginx"

We now have an "nginx" bundle.